factory/cluster.go

```diff
@@ -79,18 +88,15 @@ func DestroyCluster(g GinkgoTInterface) (string, error) {
 		return "", err
 	}
 
-	if cfg.Product == "rke2" {
-		varDir, err = filepath.Abs(shared.BasePath() + "/distros-test-framework/config/rke2.tfvars")
-	} else if cfg.Product == "k3s" {
-		varDir, err = filepath.Abs(shared.BasePath() + "/distros-test-framework/config/rke2.tfvars")
+	if cfg.Product == "rke2" || cfg.Product == "k3s"{
```

Comment:
Good thought... Simply running it would work, doing this in addtion to handle a case where it wont be either or those product, which is less likely to happen

Reply:
Updated thanks @aganesh-suse